pkg/helpers/tokencmd/request_token.go

```diff
@@ -225,13 +224,27 @@ func (o *RequestTokenOptions) RequestToken() (string, error) {
 		defer resp.Body.Close()
 
 		if resp.StatusCode == http.StatusUnauthorized {
-			if resp.Header.Get("WWW-Authenticate") != "" {
+			if len(resp.Header.Get("WWW-Authenticate")) > 0 {
 				if !o.Handler.CanHandle(resp.Header) {
 					return "", apierrs.NewUnauthorized("unhandled challenge")
 				}
 				// Handle the challenge
 				newRequestHeaders, shouldRetry, err := o.Handler.HandleChallenge(requestURL, resp.Header)
 				if err != nil {
+					if IsBootstrapPromptErr(err) {
+						tokenPromptErr := apierrs.NewUnauthorized("")
+						klog.V(4).Infof("%v", err)
+						msg := fmt.Sprintf("You must obtain an API token by visting %s/request\n", o.OsinConfig.TokenUrl)
+						details, err := ioutil.ReadAll(resp.Body)
+						if err == nil && len(details) == 0 {
```

Comment:
details are unused

Reply:
Don't need to read Body at all, all that's needed is whether err is bootstrapPromptErr.
Updated.